### ISSUE TITLE
Static site generation creates duplicate content

### ISSUE THREAD

**Issue by greatislander** [inclusive-design/wecount.inclusivedesign.ca#39]
## Describe the bug

A number of pages are duplicated when building the site statically using the `nuxt generate` command.

## To Reproduce

1. Run `npm run generate`.
2. Examine the output.
3. Note the presence of duplicate content (`/About/` and `/news/about/`, `/news/` and `/news/news/`, the main `index.html` page, `/news/we-count/`, and `we-count/` — this last produces the generate error identified in #37):

```
├── About
│   └── index.html
├── index.html
├── news
│   ├── about
│   │   └── index.html
│   ├── index.html
│   ├── news
│   │   └── index.html
│   ├── the-importance-of-peripheral-vision
│   │   └── index.html
│   ├── walrus-talk
│   │   └── index.html
│   └── we-count
│       └── index.html
└── we-count
    └── index.html
```

## Expected behavior

Site structure should be as follows:

```
├── about
│   └── index.html
├── index.html
├── news
    ├── index.html
    ├── the-importance-of-peripheral-vision
    │   └── index.html
    ├── walrus-talk
        └── index.html
```

## Additional context

I expect this is caused by unnecessary templates in `/pages`.

**Comment by TeddTech** [comment]
Hi Ned,

This is corrected in the styling PR. We can probably figure it out tomorrow

**Comment by greatislander** [comment]
Resolved in #67.
